storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java

```diff
@@ -441,21 +441,29 @@ public void notifyAboutChanges(@NonNull Changes changes) {
         }
 
         private void notifyAboutPendingChangesIfNotInTransaction() {
-            final Set<Changes> changesToSend;
+            final Set<Changes> changesSet;
```

Comment:
Yes, but what name will be used for constructed changes which will be really for sending?